examples/fee_analysis.ipynb

```diff
@@ -279,52 +290,144 @@
         "ax[current_plot].xaxis.set_ticklabels([])\n",
         "ax[current_plot].legend(fontsize=18)\n",
         "\n",
-        "# price per share & spot price\n",
+        "### direction parameters ###\n",
```

Comment:
Was this for Will's latest plot? If so, can this stuff go into a new notebook?

Reply:
good catch yes... removing them there

will's charts are in there is a separate notebook: examples/rate_scenarios.ipynb